pkg/cmd/server/admin/default_certs.go

```diff
@@ -81,6 +81,23 @@ func DefaultMasterEtcdClientCertInfo(certDir string) ClientCertInfo {
 	}
 }
 
+func DefaultNodeBootstrapClientCerts(certDir string) []ClientCertInfo {
+	return []ClientCertInfo{
+		DefaultNodeBootstrapClientCertInfo(certDir),
+	}
+}
+func DefaultNodeBootstrapClientCertInfo(certDir string) ClientCertInfo {
+	return ClientCertInfo{
+		CertLocation: configapi.CertInfo{
+			CertFile: path.Join(certDir, "node-bootstrap.crt"),
+			KeyFile:  path.Join(certDir, "node-bootstrap.key"),
+		},
+		UnqualifiedUser: "node-bootstrap",
+		User:            "system:serviceaccount:openshift-infra:node-bootstrap",
```

Comment:
This is created automatically on a new cluster - you would need to rebuild the image if you haven't.